pkg/storage/unified/sql/db/migrations/resource_mig.go

```diff
@@ -97,5 +98,11 @@ func initResourceTables(mg *migrator.Migrator) string {
 		}
 	}
 
+	// Add
+	// add column to store creator of a dashboard
+	mg.AddMigration("Add column previous_resource_version in resource_history", migrator.NewAddColumnMigration(resource_history_table, &migrator.Column{
```

Comment:
should we add this to the resource table as well to keep them consistent, or do we want to allow them to start drifting apart?

Reply:
Doesn't cost much to add both. Will do

Reply:
I am now storing those as well.